### ISSUE TITLE
Setting reduce_zero_label and other parameters in jupyter notebook config

### ISSUE THREAD

**Issue by jhaggle** [open-mmlab/mmsegmentation#2224]
I am following this tutorial and use my own custom data-set with two classes plus the background class:
https://github.com/open-mmlab/mmsegmentation/blob/master/demo/MMSegmentation_Tutorial.ipynb

I want to experiment with e.g. set things as reduce_zero_label, ignore_index, loss weight etc. How to set that in the config when running the jupyter notebook tutorial? Or should it be set elsewhere?

**Comment by xiexinch** [comment]
Hi @jhaggle,
You might take a look at our documentation. https://mmsegmentation.readthedocs.io/en/latest/tutorials/config.html
`reduce_zero_label` is a parameter of the Dataset and LoadAnnotation pipeline. https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L90
https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/pipelines/loading.py#L95
`ignore_index` is a parameter of the Dataset and DecodeHead. https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L89 https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/decode_heads/decode_head.py#L75
`loss_weight` is a parameter to your loss function. https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/losses/cross_entropy_loss.py#L209

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
